scripts/mention-bot.py

```diff
@@ -0,0 +1,811 @@
+#!/usr/bin/env python3
+"""Automatic config maker for mention-bot by facebook.
+
+Made for ver. a7ed44ed1383416f463c163b3086e9562f0e1a4a
+"""
+
+from os.path import join, expanduser, expandvars, exists
+from distutils.util import strtobool
+import sys
+import json
+import argparse
+import time
+
+PULP_PATH = "~/devel"
+PULP_REPOS = ['crane', 'pulp', 'pulp_docker', 'pulp_ostree', 'pulp_puppet', 'pulp_python',
```

Comment:
Currently the `devel` doesn't have a mention-bot set up. Honestly I don't know how much PRs are against this repo if it's worth it. But I think it might be good to do so. I think @seandst could more reliably answer if it's worth it.